### PR TITLE
[SIMD] Add the AArch64 support for neon binary macros

### PR DESCRIPTION
**Pull request by dongju-chae**
This PR adds the AArch64 support for neon binary macros.

Signed-off-by: Dongju Chae <dongju.chae@samsung.com>

### CODE REVIEW

File: orc/orcrules-neon.c [hunk 1]
```diff
@@ -845,85 +1116,178 @@ neon_rule_loadX (OrcCompiler *compiler, void *user, OrcInstruction *insn)
...
+        shift = 0;
+      }
+      ORC_ASM_CODE(compiler,"  vld1.%d %s[0], [%s]%s\n",
+          8<<shift,
+          orc_neon_reg_name (dest->alloc),
+          orc_arm_reg_name (ptr_register),
+          update ? "!" : "");
+      code = 0xf4a0000d;
+      code |= shift<<10;
+      code |= (0&7)<<5;
```
Comment (myungjoo) [inline orc/orcrules-neon.c]: Typo? ```(0&7)```

File: orc/orcrules-neon.c [hunk 2]
```diff
@@ -935,86 +1299,178 @@ neon_rule_storeX (OrcCompiler *compiler, void *user, OrcInstruction *insn)
...
+        shift = 0;
+      }
+      ORC_ASM_CODE(compiler,"  vst1.%d %s[0], [%s]%s\n",
+          8<<shift,
+          orc_neon_reg_name (src->alloc),
+          orc_arm_reg_name (dest->ptr_register),
+          update ? "!" : "");
+      code = 0xf480000d;
+      code |= shift<<10;
+      code |= (0&7)<<5;
```
Comment (myungjoo) [inline orc/orcrules-neon.c]: ```(0&7)```?

File: orc/orcrules-neon.c (hunk 1 above)
Comment (dongju-chae) [reply orc/orcrules-neon.c]: ohh.. thanks.

Comment (dongju-chae) [reply orc/orcrules-neon.c]: Oh nono. It's not my code.

Comment (dongju-chae) [reply orc/orcrules-neon.c]: Hmm I'm not sure about its intention..?

**Pull request merged**